package.json

```diff
@@ -70,13 +71,15 @@
     "resolve": "^1.0.0",
     "send": "^0.14.1",
     "spdy": "^3.3.3",
-    "ua-parser-js": "^0.7.12"
+    "ua-parser-js": "^0.7.12",
+    "ws": "^1.1.1"
   },
   "devDependencies": {
     "@types/assert": "0.0.29",
     "@types/chai": "^3.4.34",
     "@types/chai-as-promised": "0.0.29",
     "@types/lru-cache": "^2.5.32",
+    "@types/chokidar": "=1.4.29",
```

Comment:
For more discussion on this specific version see https://github.com/DefinitelyTyped/DefinitelyTyped/pull/13126#discussion_r92944133